### PR TITLE
[#1769] Editor tests can now fuction without the ui enabled

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1769-core-tests-failing

### CODE REVIEW

File: src/editor/module.js [hunk 1]
```diff
@@ -113,9 +113,7 @@ define( [ "core/eventmanager", "core/trackevent", "./editor",
      */
     this._start = function( onModuleReady ){
       onModuleReady();
-      if( butter.config.value( "ui" ).enabled !== false ){
-        butter.ui.areas.editor = new butter.ui.Area( "editor-area", _editorAreaDOMRoot );
-        _toggler = new Toggler( function( e ) {
+      _toggler = new Toggler( function( e ) {
           var newState = !_editorAreaDOMRoot.classList.contains( "minimized" );
```
Comment (jbuck) [inline src/editor/module.js]: Adjust the indent for this code block then you are :ok_hand:

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e68f516a32dc0d1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/43dd4f2e926ec27/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e68f516a32dc0d1/output.txt

Total script time: 0.77 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/43dd4f2e926ec27/output.txt

Total script time: 0.49 mins

Server is running on http://pr645.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/40526e15c002f66/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/40526e15c002f66/output.txt

Total script time: 0.64 mins

**Pull request merged**